vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/DashboardWidget.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.dashboard;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+
+/**
+ * @author Vaadin Ltd
+ */
+@Tag("vaadin-dashboard-widget")
+@JsModule("@vaadin/dashboard/src/vaadin-dashboard.js")
+@NpmPackage(value = "@vaadin/dashboard-widget", version = "24.5.0-alpha8")
```

Comment:
There will assumably be no separate package for `dashboard-widget`

Reply:
Same comments as here https://github.com/vaadin/flow-components/pull/6548#discussion_r1718042354

Reply:
Updated package name and version.